test/integration/conftest.py

```diff
@@ -655,3 +655,21 @@ def as_db(request: 'FixtureRequest', autosubmit: Autosubmit, tmp_path: 'LocalPat
     autosubmit_exp('____')
 
     return backend
+
+
+def sleep(timeout, retry=3):
```

Comment:
Maybe rename to timeout? Or wait_something? But you got the gist of it very quickly! 💪 👍